### PR TITLE
[BLOG] Add Captioning: A Newcomer's Guide

### PR DESCRIPTION
**Pull request by isabela-pf**
This adds a blog post about what I learned in coordinating captioning services for virtual events. Some notes:

- ~~I intend to add an image, but didn't want it to block the review of the text so I have not done it yet.~~ This has now been added.
- I'm not confident in the categories I chose for this post, so feedback there is also helpful.

Please let me know if you have any questions and what you'd like me to change. Thank you! 🌻 

---

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/captioning-a-newcomers-guide.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+As mentioned in *Why should I caption things?* the [W3C has also complied a list of business case arguments in favor of accessibility](https://www.w3.org/WAI/business-case/) accommodations such as captions.
+
+### Are there criteria for this? 
+
+As far as I know, there is not a singular standard for captioning content, file format, and styling that are used absolutely everywhere. However, there are only a few dominant ones. Once again, [WCAG covers this as “alternatives for time-based media”](https://www.w3.org/TR/WCAG22/#time-based-media) and navigating to the [Understanding articles](https://www.w3.org/WAI/WCAG22/Understanding/time-based-media.html) per guideline will provide you with examples of how and how not to caption.
+
+A few more tips: 
+
+* Follow the file format or other formatting relevant to where you are hosting the content. This ensures it behaves as expected. If you are hiring a captioner, know what service you will be hosting onahead of time so both sides can check they have the appropriate format and tech support.
```
Comment (rgommers) [inline apps/labs/posts/captioning-a-newcomers-guide.md:96]:
```suggestion
* Follow the file format or other formatting relevant to where you are hosting the content. This ensures it behaves as expected. If you are hiring a captioner, know what service you will be hosting on ahead of time so both sides can check they have the appropriate format and tech support.
```

File: apps/labs/posts/captioning-a-newcomers-guide.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+* If you know someone in your community who had a good experience with a captioner, ask if they’d be willing to introduce you or pass on the contact info. The bonus is that these captioners might have more familiarity with the terms used in your community and better know how to caption them. I bring this up because this was really tricky with the many software homophones (and affinity for switching out or removing vowels in tech names) in captioning for events in software and scientific spaces.
+* Make time to do a dry run of your event. Ask the if the captioner is interested in being involved in it and take the time to test out the tech support aspects. If they say they are not interested, accept that.
+* Plan time ahead of your event to get the captioner set up and running. Test it before you have attendees watching.
+* Ask your captioner they want the event materials ahead of time, and if so how much ahead of time? If they say they do not want materials ahead of time, accept that. This is likely to depend on the captioner’s preferences and their comfort level with the vocabulary in your field, so it’s best to let them decide but still good to offer.
```
Comment (rgommers) [inline apps/labs/posts/captioning-a-newcomers-guide.md:145]:
```suggestion
* Ask your captioner if they want the event materials ahead of time, and if so how much ahead of time? If they say they do not want materials ahead of time, accept that. This is likely to depend on the captioner’s preferences and their comfort level with the vocabulary in your field, so it’s best to let them decide but still good to offer.
```

File: apps/labs/posts/captioning-a-newcomers-guide.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+* Word of mouth was by far the most helpful for finding reliable live captioning. Start asking people you have who run events. If a captioner replies they aren’t available, you may want to ask if they recommend anyone for you to contact instead.
+* If you know someone in your community who had a good experience with a captioner, ask if they’d be willing to introduce you or pass on the contact info. The bonus is that these captioners might have more familiarity with the terms used in your community and better know how to caption them. I bring this up because this was really tricky with the many software homophones (and affinity for switching out or removing vowels in tech names) in captioning for events in software and scientific spaces.
+* Make time to do a dry run of your event. Ask the if the captioner is interested in being involved in it and take the time to test out the tech support aspects. If they say they are not interested, accept that.
```
Comment (rgommers) [inline apps/labs/posts/captioning-a-newcomers-guide.md:143]:
```suggestion
* Make time to do a dry run of your event. Ask if the captioner is interested in being involved in it and take the time to test out the tech support aspects. If they say they are not interested, accept that.
```

**Review by rgommers — APPROVED** [review]
This is a very nice post, thanks @isabela-pf!

I have very few comments, only a few typo fixes - not because I didn't read carefully, but because it's a very polished blog post already.

> I'm not confident in the categories I chose for this post, so feedback there is also helpful.

`Access-centered` and `OSS experience` seem appropriate to me. The only other existing tag I'd consider is `Community`, but it overlaps with `OSS experience` so choosing just that one seems perfectly fine.

I'll commit the typo fixes and then will hit the green button here. Nice work!

**Comment by isabela-pf** [comment]
Thanks for making these changes. I am ready for it to go whenever, I just set it further ahead for review and because I don't know what the blog post calendar currently looks like.

If you have a date you'd like me to change it to, let me know what date and I can change that.

**Comment by rgommers** [comment]
Thanks @isabela-pf. It doesn't matter to me, the sooner the better I'd say. @pavithraes or @bskinn maybe you know when the next release is planned?

**Comment by bskinn** [comment]
@rgommers We're on an as-requested cadence for site builds lately, no set schedule. 

So--I'll work with @pavithraes or @gabalafou to get this live either tomorrow or Wednesday! I gave the post a quick read, and it all looks good to me.

(Attn @Katebrack, for prep for socials posts once it's live.)

File: apps/labs/posts/captioning-a-newcomers-guide.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
+---
+title: 'Captioning: A Newcomer’s Guide'
+published: February 1, 2024
```
Comment (bskinn) [inline apps/labs/posts/captioning-a-newcomers-guide.md:3]: +/- a day won't matter much. By the time this goes live it'll be Wednesday somewhere on Earth, anyways, so this'll be accurate either day it goes live.

```suggestion
published: January 24, 2024
```

**Comment by rgommers** [comment]
Sounds good, thanks Brian!

**Comment by isabela-pf** [comment]
Thank you all for the review and help!

**Pull request merged**